packages/@datapull/chef-origin/index.js

```diff
@@ -0,0 +1,84 @@
+const util = require('util');
+const objectPath = require('object-path');
+const chef_api = require("chef-api");
+
+class ChefOrigin {
+
+    constructor(config) {
+        this.config = config;
+    }
+
+    get originDeclaration() {
+        return {
+            name: ['Chef', this.config.organization].join(':'),
+            runner: this.pullData.bind(this)
+        };
+    }
+
+    async pullData(config) {
+        console.log('[Chef] pulling data for ', config.searchName);
+
+        const options = {
+            user_name: config.username,
+            key_path: config.key_path,
+            url: config.apiUrl
+        };
+
+        const chef = new chef_api();
+        chef.config(options);
+        const nodes = await util.promisify(chef.getNodes)()
+        const results = [];
+        for (const nodeName in nodes) {
+            if (nodes.hasOwnProperty(nodeName)) {
+                const node = await util.promisify(chef.getNode)(nodeName);
+                const info = {
```

Comment:
please return the complete node information from this origin, in this way we can specify the necessary fields in the pipeline's yaml